mlfinlab/tests/test_filters.py

```diff
@@ -51,3 +51,31 @@ def test_cusum_filter(self):
                     cusum = max(np.abs(sneg), spos)
 
                     self.assertTrue(cusum >= threshold)
+
+    def test_dynamic_cusum_filter(self):
```

Comment:
Are there any extra edge cases you want us to test?

Reply:
The first part of test (I mean yours) tests edge cases, I've decided that for dynamic part simply testing the resulting shape would be enough